build/buildall

```diff
@@ -160,8 +160,15 @@ shift
 
 done
 
+case `uname -m` in
+    aarch64|arm64)
+        cpu_arch='arm64';;
+    *)
```

Comment:
I would rather we be specific here and fail the script if an unrecognized machine hardware name is reported.  Comment applies to all the places where this logic was copied.

Reply:
yes, `cpu_arch='amd64'` could simply be the script default. but we should exit error if non-supported archs

Reply:
fixed